dali/ft/ftbase.cpp

```diff
@@ -119,15 +121,20 @@ void PartitionPoint::deserialize(MemoryBuffer & in)
     in.read(whichSlave);
     modifiedTime.deserialize(in);
     ::deserialize(in, fixedText);
+    if (in.remaining()>0)
+    {
```

Comment:
I don't think these fields are actually needed on the slave - so it may be better to not serialize them, and have them default initialised to an invalid number.  Solves the serialization problem....

... see further comments above you can remove them completely from this class instead.